scripts/windows/conda_trim.ps1

```diff
@@ -0,0 +1,15 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
```

Comment:
Is conda_trim.ps1 being called? If so, for my understanding, could you please point to where it is called?

Reply:
Oh sorry. I was initially concerned about the bundle size, so i wanted to trim not required packaged. However currently the fetch performance is fine so i never actually added anything to remove.